server/lists/index.js

```diff
@@ -31,6 +31,12 @@ module.exports = {
             type: listService.types.DYNAMIC,
             adapter: teamsAdapter
         },
+        ALL_TEAMS: {
+            client: 'INFO',
+            endpoint: '/team/all',
```

Comment:
I'm wondering if to be more restful, we should just have a `GET` on `/team` return a list of teams? Not too fussed about it being in this change though...